iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -254,47 +251,21 @@ static Optional<Catalog> loadCatalog(Configuration conf, String catalogName) {
    * @param catalogType type of the catalog
    * @return complete map of catalog properties
    */
-  private static Map<String, String> getCatalogProperties(Configuration conf, String catalogName, String catalogType) {
+  private static Map<String, String> getCatalogProperties(
+      Configuration conf, String catalogName, String catalogType) {
     Map<String, String> catalogProperties = Maps.newHashMap();
     conf.forEach(config -> {
       if (config.getKey().startsWith(InputFormatConfig.CATALOG_DEFAULT_CONFIG_PREFIX)) {
         catalogProperties.putIfAbsent(
-            config.getKey().substring(InputFormatConfig.CATALOG_DEFAULT_CONFIG_PREFIX.length()),
-            config.getValue());
+                config.getKey().substring(InputFormatConfig.CATALOG_DEFAULT_CONFIG_PREFIX.length()),
+                config.getValue());
       } else if (config.getKey().startsWith(InputFormatConfig.CATALOG_CONFIG_PREFIX + catalogName)) {
```

Comment:
could we please extract `String keyPrefix = InputFormatConfig.CATALOG_CONFIG_PREFIX + catalogName;` so we won't generate new String objects on each iteration

Reply:
Done

Reply:
Fixed.